packages/react-components/src/components/ControlBarButton.tsx

```diff
@@ -115,12 +135,20 @@ export const ControlBarButton = (props: ControlBarButtonProps): JSX.Element => {
     );
   }, [labelText, props?.labelKey, props?.styles?.label, theme]);
 
+  const tooltipContent =
+    props?.strings?.tooltipContent ??
+    (props?.checked ? props?.strings?.tooltipOnContent : props?.strings?.tooltipOffContent);
+
+  const tooltipId = props.tooltipId ?? props.labelKey ? props.labelKey + '-tooltip' : undefined;
+
   return (
-    <DefaultButton
-      {...props}
-      styles={componentStyles}
-      onRenderText={props.showLabel ? props.onRenderText ?? DefaultRenderText : undefined}
-      onRenderIcon={props.onRenderIcon ?? DefaultRenderIcon}
-    />
+    <ControlButtonTooltip content={tooltipContent} id={tooltipId}>
```

Comment:
Is it possible for develoers to disable tooltips? I guess some might wanna do it

Reply:
They are disabled by default and only show if a `tooltipContent` string is supplied